payroll/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "Payroll",
-    "version": "14.0.5.3.2",
+    "version": "14.0.5.4.0",
```

Comment:
Please remove this. It will be done automatically when the PR is merged.

Reply:
Following the oca guide, if you change the structure of the object, you have to make a version change in the y... but I don't know if you have to keep the z.

Reply:
When the ocabot merges your changes it will bump the version as necessary. You don't have to worry about this.